compiler/passes/src/const_propagation/statement.rs

```diff
@@ -54,7 +54,9 @@ impl StatementReconstructor for ConstPropagationVisitor<'_> {
     }
 
     fn reconstruct_assign(&mut self, assign: AssignStatement) -> (Statement, Self::AdditionalOutput) {
```

Comment:
Does the comment in `const_propogation/mod.rs` need updating?

Reply:
This pass is now also transforming assignment locations right?
The comment describing the pass might be out of date.